request-controller/src/main/java/org/wildfly/extension/requestcontroller/RequestControllerRootDefinition.java

```diff
@@ -59,6 +62,10 @@
             .build();
     public static final RequestControllerRootDefinition INSTANCE = new RequestControllerRootDefinition(true);
 
+    static final RuntimeCapability<Void> REQUEST_CONTROLLER_CAPABILITY =
+            RuntimeCapability.Builder.of(REQUEST_CONTROLLER_CAPABILITY_NAME, false, RequestController.class)
+                    .build();
+
     private final boolean registerRuntimeOnly;
 
     RequestControllerRootDefinition(boolean registerRuntimeOnly) {
```

Comment:
While you are here, 
can you also register capability on resource so we have info of what resource provides what.
it is a new api not used much yet but everything will soon move to this, as it is needed to make proper model reference impl.

just add 
@Override
    public void registerCapabilities(ManagementResourceRegistration resourceRegistration) {
        resourceRegistration.registerCapability(REQUEST_CONTROLLER_CAPABILITY);
    }